starwars/src/components/CardImage.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './CardStyles.css';
+
```

Comment:
I like that you went beyond the mvp to add in the images from the provided api!!